launch_testing/setup.py

```diff
@@ -0,0 +1,39 @@
+import glob
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+setup(
+    name='launch_testing',
+    version='0.7.3',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages', ['resource/launch_testing']),
+        ('lib/launch_testing', glob.glob('example_processes/**')),
+        ('share/launch_testing/examples', glob.glob('examples/[!_]**')),
+    ],
+    entry_points={
+        'console_scripts': ['launch_test=launch_testing.launch_test:main']
+    },
+    install_requires=['setuptools'],
+    zip_safe=True,
+    author='Pete Baughman',
```

Comment:
This should be consistent with the author in `package.xml`

Reply:
Done.